packages/shared/src/components/filters/FeedFilters.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { ReactElement, useEffect, useState } from 'react';
+import classNames from 'classnames';
+import sizeN from '../../../macros/sizeN.macro';
+import FilterMenu from './FilterMenu';
+import { ClickableText } from '../buttons/ClickableText';
+import FilterIcon from '../../../icons/arrow.svg';
+
+const asideWidth = sizeN(89);
+
+interface FeedFiltersProps {
+  isOpen?: boolean;
+  onBack?: () => void;
+}
+
+let timeout: NodeJS.Timeout;
```

Comment:
This is a better typing as you currently rely on a node runtime which is actually not the right one.
Our runtime is the browser not node.

On another note, you should not define global variables for hooks. You need to use `useRef` for this.
There are many examples in this codebase and on the internet on how to use `setTimeout` with React Hooks. Give it a look, not that trivial.

```suggestion
let timeout: ReturnType<typeof setTimeout>;
```

Reply:
I'm gonna look it up! Thank you for this!